templates/enterprise-landing-zone/example.tfvars

```diff
@@ -18,7 +18,6 @@ enable_compartment_delete  = false
 
 # security
 enable_cloud_guard                           = true
-cloud_guard_target_tenancy                   = false
```

Comment:
Why do we remove this variable from the tfvars?

Reply:
If it's set to true, the deployment fails because each environment will try to deploy a cloud guard target to the tenancy and an error will be thrown that the tenancy already has cloud guard attached. I'm not even sure we need it.

Reply:
Right, if we don't need this variable we can directly remove it. Or we can add to a document that if setting the value to true, the customer should only have one landing zone set the value to true, otherwise it will have conflict. I do think setting the value to true would be a use case.

Reply:
@yupeiyang-oci I think it's best to remove it, and I can update the PR too.